_disability-benefits/apply-for-benefits.md

```diff
@@ -22,30 +22,23 @@ relatedlinks:
     description: ""
 ---
 
-<div class="main interior" role="main" markdown="0">
-<div class="section one" markdown="0">
-<div class="row" markdown="0">
-<div class="small-12 columns"  markdown="1">
-
 ### Who should read this?
```

Comment:
@webinista take a look at these two and compare:

https://vets-website-pr-2401.herokuapp.com/disability-benefits/apply-for-benefits/
https://www.vets.gov/disability-benefits/apply-for-benefits/

I think some of the breathing room in the design got lost in this change. (sorry this is a bit of a nit!)